common/src/main/java/org/opensearch/ml/common/connector/HttpConnector.java

```diff
@@ -127,6 +129,9 @@ public HttpConnector(String protocol, XContentParser parser) throws IOException
                 case CLIENT_CONFIG_FIELD:
                     connectorClientConfig = ConnectorClientConfig.parse(parser);
                     break;
+                case MODEL_INTERFACE_FIELD:
+                    modelInterface =  getInterfaceMap(parser.map());
+                    break;
```

Comment:
Just want to double confirm that we don't have to version control for it, right?